ElementX/Sources/Services/Room/RoomSummary/RoomSummary.swift

```diff
@@ -81,6 +84,17 @@ extension RoomSummary: CustomStringConvertible {
             return alias
         }
         
+        guard heroes.count > 0 else {
+            return ""
+        }
+        
+        if activeMembersCount > 0 {
+            var components = heroes.compactMap(\.displayName)
+            components.append(L10n.commonManyMembers(Int(activeMembersCount)))
```

Comment:
Shouldn't this be something like
```suggestion
            let othersCount = Int(activeMembersCount) - heroes.count
            if othersCount > 0 {
                components.append(L10n.commonManyMembers(othersCount))
            }
```